.eslintrc.json

```diff
@@ -31,7 +31,7 @@
   "rules": {
     // Best Practices
     "no-var": "error",
-    "no-unused-vars": ["error", { "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }],
+    "no-unused-vars": ["error", { "args": "none", "varsIgnorePattern": "^_", "argsIgnorePattern": "^_" }],
```

Comment:
I am setting `no-unused-vars` checking on args to none as this one does not make a lot of sense and usually leads to a lot of errors.
https://eslint.org/docs/rules/no-unused-vars#args

Reply:
I dont think we should remove it :). I am fine with not make a hard fail to merge PR but it is very useful to have.

Reply:
I should have explained the root cause behind this one more. The `no-unused-vars` checking is good to have but it is not working as expected on function arguments. For example, when `args: none` is not provided to this check; the linter returns the following error:
```
/Users/kunalnan/Documents/AWS/amazon-chime-sdk-component-library-react/src/types/index.ts
  20:1   error  This line has a length of 128. Maximum allowed is 120                                      max-len
  32:23  error  'isEnabled' is defined but never used. Allowed unused args must match /^_/u                no-unused-vars
  34:29  error  'hasReached' is defined but never used. Allowed unused args must match /^_/u               no-unused-vars
  40:24  error  'sourceId' is defined but never used. Allowed unused args must match /^_/u                 no-unused-vars
  45:3   error  'Loading' is defined but never used. Allowed unused vars must match /^_/u                  no-unused-vars
  46:3   error  'Succeeded' is defined but never used. Allowed unused vars must match /^_/u                no-unused-vars
  47:3   error  'Failed' is defined but never used. Allowed unused vars must match /^_/u                   no-unused-vars
  48:3   error  'Ended' is defined but never used. Allowed unused vars must match /^_/u                    no-unused-vars
  49:3   error  'JoinedFromAnotherDevice' is defined but never used. Allowed unused vars must match /^_/u  no-unused-vars
  50:3   error  'Left' is defined but never used. Allowed unused vars must match /^_/u                     no-unused-vars
  51:3   error  'TerminalFailure' is defined but never used. Allowed unused vars must match /^_/u          no-unused-vars
  65:3   error  'UNSET' is defined but never used. Allowed unused vars must match /^_/u                    no-unused-vars
  66:3   error  'IN_PROGRESS' is defined but never used. Allowed unused vars must match /^_/u              no-unused-vars
  67:3   error  'GRANTED' is defined but never used. Allowed unused vars must match /^_/u                  no-unused-vars
  68:3   error  'DENIED' is defined but never used. Allowed unused vars must match /^_/u                   no-unused-vars
  72:3   error  'None' is defined but never used. Allowed unused vars must match /^_/u                     no-unused-vars
  73:3   error  'Audio' is defined but never used. Allowed unused vars must match /^_/u                    no-unused-vars
  74:3   error  'Video' is defined but never used. Allowed unused vars must match /^_/u                    no-unused-vars
  75:3   error  'AudioAndVideo' is defined but never used. Allowed unused vars must match /^_/u            no-unused-vars
```
And the error: variable is defined and never used errors are coming from this `type.ts` file: [index.ts](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/main/src/types/index.ts).
 As you can see from the error, a lot of these errors are not really valid. So, ignoring the check of function arguments helps get rid of the noise. 


> the argsIgnorePattern rule should get applied asking to underscore the variable right?

The `argsIgnorePattern` is really sort of separate from the linter noise that I am trying to remove.

Reply:
I think, the `argsIgnorePattern` is actually kicking in here and providing us the error with "Allowed unused vars must match /^_/u".

So, would it be better to exclude the `types` file from linting over adding "none"?